lib/node/rpc.js

```diff
@@ -2072,14 +2072,17 @@ class RPC extends RPCBase {
       };
     }
 
+    const isWitness = addr.isProgram();
     const script = Script.fromAddress(addr);
 
     return {
```

Comment:
Similar to previous comment, the `undefined` properties could be left off the object (even without `JSON.stringify` dropping them) by doing:

```js
if (isWitness) {
  result.witness_version = addr.version;
  result.witness_program = addr.hash.toString('hex');
}
```